src/modules/events/forms/EventForm.tsx

```diff
@@ -53,6 +53,14 @@ const initialData = {
   metaData: {},
 };
 
+function getFrequency(data: string) {
```

Comment:
There is an actual table that tracks the frequency, please reach out to herbert on how to get that data...